### DISCUSSION TITLE
Accept Monero as payment for Ente subscriptions.

### DISCUSSION THREAD

**Discussion by donttracemebruh** [ente-io/ente#584] [Enhancements]
Monero (getmonero.org) is a decentralized open source digital cash network that prioritizes it's users' privacy and security. Monero aligns with Ente's vision of open source privacy-preserving software. Many users could benefit from a private way to pay for their Ente subscriptions that does not depend on a centralized fiat payment entity which collects customer data and leaves that data vulnerable to compromise.

Monero can be relatively easily integrated as a payment option for Ente with BTCPay Server (https://github.com/btcpayserver), an open source payment processor. A setup guide for Monero on BTCPay Server can be found here: https://sethforprivacy.com/guides/accepting-monero-via-btcpay-server/

**Comment by aquaspy** [comment]
I would really enjoy this option
